pydis_site/apps/content/utils.py

```diff
@@ -0,0 +1,117 @@
+import os
+from pathlib import Path
+from typing import Dict, List, Optional, Union
+
+import requests
+import yaml
+from dateutil import parser
+from django.conf import settings
+from django.http import Http404
+from markdown2 import markdown
+
+COMMITS_URL = "https://api.github.com/repos/{owner}/{name}/commits?path={path}&sha={branch}"
+BASE_ARTICLES_LOCATION = "pydis_site/apps/content/resources/content/"
+
+
+def _get_base_path() -> Path:
+    """Have extra function for base path getting for testability."""
+    return Path(settings.BASE_DIR, "pydis_site", "apps", "content", "resources", "content")
```

Comment:
I don't see why there's a method for this - wouldn't it be worth moving the `Path` object to a constant in settings, and patching `django.conf.settings.CONTENT_BASE_PATH`?